pkg/apis/serving/fieldmask.go

```diff
@@ -57,6 +58,7 @@ func VolumeSourceMask(in *corev1.VolumeSource) *corev1.VolumeSource {
 	out.Secret = in.Secret
 	out.ConfigMap = in.ConfigMap
 	out.Projected = in.Projected
+	out.EmptyDir = in.EmptyDir
```

Comment:
Any reason we don't check the feature flag here like we do for example for the SecurtityContext types?

Reply:
A few lines above before a volumeValidation call is done we check the features flags and emit an error.
Creating more errors from that point on seemed verbose to me. Otherwise we could create two errors (volumeMask and volumeSourceMask) and avoid the top one.